test/simplecov_test_helper.rb

```diff
@@ -9,9 +9,3 @@
 SIMPLECOV_MIN_RUBY_VERSION = '2.7.0'
 
 require 'simplecov' if RUBY_VERSION >= SIMPLECOV_MIN_RUBY_VERSION
```

Comment:
I decided to leave this file so that we have a single location where the simplecov minimum version constraint is enforced.